docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -13,10 +13,14 @@ This page explains how to get started monitoring the security posture of your cl
 * The CSPM integration is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
 * CSPM only works in the `Default` {kib} space. Installing the CSPM integration on a different {kib} space will not work. 
 * CSPM is supported only on AWS, GCP, and Azure commercial cloud platforms, and AWS GovCloud. Other government cloud platforms are not supported. https://github.com/elastic/kibana/issues/new/choose[Click here to request support].
-* To view posture data, you need `read` privileges for the following {es} indices:
+* `read` privileges for the following {es} indices:
 ** `logs-cloud_security_posture.findings_latest-*`
 ** `logs-cloud_security_posture.scores-*`
 ** `Logs-cloud_security_posture.findings`
+* The following {kib} privileges:
+** Security: `read`
```

Comment:
Super tiny nit since the priv options are capitalized in the UI. Feel free to reject this edit if you don't think the case needs to match what users see in the UI. If you do, it should be applied across all of the applicable areas on this and the other pages. 

```suggestion
** Security: `Read`
```